apisix/plugins/ldap-auth.lua

```diff
@@ -152,7 +152,7 @@ function _M.rewrite(conf, ctx)
         create_consumer_cache, consumer_conf)
     local consumer = consumers[userdn]
     if not consumer then
-        return 401, {message = "Invalid API key in request"}
+        return 401, {message = "Invalid password or username"}
```

Comment:
```suggestion
        return 401, {message = "Invalid user authorization"}
```

Reply:
And you should update https://github.com/apache/apisix/pull/6707/files#diff-8e5b15fc718c151b0f2b2fbf464e510f31c3e0bf31a350aa2f5ca01349f4a2f6R149 too

Reply:
done. unify to " Invalid user authorization"